_posts/2018-07-30-ARIA-properties.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: ARIA Properties
+description: A quick introduction to ARIA's property system.
+author: Alex Brenon
+categories: basics
```

Comment:
should reclassify as background or quick tips.
@ericwbailey you got thoughts on where to put this?